Dockerfile

```diff
@@ -1,5 +1,14 @@
-FROM balenalib/rpi-raspbian:stretch
+ARG DISTRO=buster
+FROM balenalib/rpi-raspbian:${DISTRO}
+
+ARG UID
+ARG GID
 
 RUN apt-get update
 RUN apt-get -y install coreutils quilt parted debootstrap zerofree pxz zip dosfstools bsdtar libcap2-bin grep rsync xz-utils file git curl kmod fakeroot device-tree-compiler dkms
-RUN apt-get -y install build-essential dh-make
+RUN apt-get -y install build-essential dh-make raspberrypi-kernel-headers bc sudo python bison flex libssl-dev
```

Comment:
Review whether all these dependencies are needed now that we are not rebuilding any in-tree module.